queries/wikidata/brad-pitt.sparql

```diff
@@ -0,0 +1,10 @@
+# Directors of movies starring Brad Pitt in Wikidata
+# Datasource: https://query.wikidata.org/bigdata/ldf
+SELECT ?movie ?title ?name
+WHERE {
+  ?movie wdt:P161 wd:Q35332;
```

Comment:
Can we look up Brad Pitt by label to make the query at least somewhat explanatory?

Reply:
Done!